### PR TITLE
Add bookmarks

### PR DESCRIPTION
**Pull request by mclintprojects**

### CODE REVIEW

**Comment by mclintprojects** [comment]
I deployed your work to Netlify and found a bug. Each idea has a unique id but it's only within an idea category. So in different idea categories we can have items with the same id.

Since you're bookmarking ideas with only id, if I bookmark an idea with id 1 in category 1 and I go to category 2, the idea there with id 1 is set as bookmarked when it shouldn't be.

You should find by both category id and idea id to fix this bug. There's no category id so I use the category's index as its id.

File: web/ideabag2-web/src/views/Bookmarks.vue [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      }
+    },
+    loadIdea(id) {
+      const categoryId = id[0];
+      const ideaId = id[3] - 1;
+      this.ideas.push(this.$store.getters.categories[categoryId].items[ideaId]);
+    }
+  },
+  components: {
+    "idea-list": IdeaList
```
Comment (mclintprojects) [inline web/ideabag2-web/src/views/Bookmarks.vue]: If you name a component like `IdeaList, NavBar, CategoryList`, Vue automatically makes the name `'idea-list', 'nav-bar', 'category-list'` so in this case you can say `components: { IdeaList }` and it'd still work fine.

**Comment by jarikmarwede** [comment]
Should be fixed now.

**Comment by mclintprojects** [comment]
> 
> 
> Should be fixed now.

All good! Merging now.

**Pull request merged**